### PR TITLE
Book Suggestion

### PR DESCRIPTION
**Pull request by franciscoabalan**
## Summary
Create model Book Suggestion

## Trellocard
https://trello.com/c/iSTaigzP/25-crear-modelo-de-booksuggestion

## Screenshot
Captura de Pantalla 2019-10-28 a la(s) 15 05 30

### CODE REVIEW

**Review by joredudiaz — APPROVED** [review]
I like it, 👏 wait for @aleochoam approval

File: app/policies/book_policy.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class BookPolicy < ApplicationPolicy
```
Comment (aleochoam) [inline app/policies/book_policy.rb]: 👀

File: app/models/user.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@ class User < ApplicationRecord
 
   # Relations
   has_many :rents, dependent: :destroy
+  has_many :book_suggestions, dependent: :destroy
```
Comment (aleochoam) [inline app/models/user.rb:11]: Let's add a  test for this new relation

Comment (aleochoam) [inline app/models/user.rb:11]: In another context, a library could maybe keep the suggestions made by old users, event if they are deleted from the db. Here is ok!

**Review by aleochoam — COMMENTED** [review]
Perfect! just one comment!

File: app/models/user.rb (hunk 2 above)
Comment (franciscoabalan) [reply app/models/user.rb:11]: Done

Comment (franciscoabalan) [reply app/models/user.rb:11]: Good

**Review by aleochoam — APPROVED** [review]

**Pull request merged**